src/redux/modules/producers/producers.js

```diff
@@ -131,3 +138,11 @@ export function create(data) {
     promise: (client) => client.post('/producers', { data }),
   };
 }
+
+/**
+ * After a new producer is created we reset
+ * in the store `createdProducer` key
+ */
+export function resetCreated() {
+  return { type: RESET_CREATED_PRODUCER };
+}
```

Comment:
why producer reducer have to store state for supplier creation? can we decouple this a bit more?